pkgs/os-specific/linux/wpa_supplicant/default.nix

```diff
@@ -48,6 +48,7 @@ stdenv.mkDerivation rec {
     CONFIG_HS20=y
     CONFIG_HT_OVERRIDES=y
     CONFIG_IEEE80211AC=y
+    CONFIG_IEEE80211AX=y
```

Comment:
Both CONFIG_IEEE80211AX=y and CONFIG_SAE=y
are already enabled by default
https://w1.fi/cgit/hostap/tree/wpa_supplicant/defconfig#n487
https://w1.fi/cgit/hostap/tree/wpa_supplicant/defconfig#n255

Reply:
You are right, but there's a lot more unnecessary assignments already in there (e.g. CONFIG_IEEE80211AC=y is also default), mostly because the upstream `defconfig` changes from time to time. I don't see an issue with explicitly including stuff like this if just to document that it was being though of.

Reply:
> CONFIG_IEEE80211AX=y

This one isn't in release 2.10, looks like CONFIG_SAE is though.

Reply:
I think that we shouldn't introduce more unnecessary assignments, if we don't do that this list is going to get bigger and bigger until one day we are going to maintain an entire defconfig. I also think that we should remove every duplicated assignments in another pr.

Reply:
@Tungsten842  I just re-checked and saw that your original your comment is wrong. You link to the wpa_supplicant defconfig, which is different from the hostapd defconfig. In hostapd almost nothing is enabled by default.

Reply:
> @Tungsten842 I just re-checked and saw that your original your comment is wrong. You link to the wpa_supplicant defconfig, which is different from the hostapd defconfig. In hostapd almost nothing is enabled by default.

I linked both of them.

Reply:
No, you linked to CONFIG_SAE and CONFIG_IEEE80211AX in [tree/wpa_supplicant](https://w1.fi/cgit/hostap/tree/). In hostapd the option is not enabled by default, [even on master](https://w1.fi/cgit/hostap/tree/hostapd/defconfig#n159)

Reply:
Ah sorry, my mistake. This is about wpa_supplicant. Forget what I said.

Reply:
Sorry again for the confusion. The redundant definitions have been removed.